include/Probe.php

```diff
@@ -57,6 +59,8 @@ private function xrd($host) {
 		$ssl_url = "https://".$host."/.well-known/host-meta";
 		$url = "http://".$host."/.well-known/host-meta";
 
+		$baseurl = "http://".$host;
```

Comment:
Why not directly do `self::$baseurl = "http://".$host;` here and removing the subsequent added line later?